notebooks/privacy_metric.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
Could you please add a link and description for this notebook to `notebooks/README.md`?